test/unit-tests/org/listing-usersTest.js

```diff
@@ -0,0 +1,152 @@
+const server = require('../../../test-utils/index')
+const chai = require('chai')
+const expect = chai.expect
+chai.use(require('chai-http'))
+
+const orgFixtures = require('./mockObjects.org')
+const orgController = require('../../../src/controller/org.controller/org.controller')
+const orgParams = require('../../../src/controller/org.controller/org.middleware')
+
+const errors = require('../../../src/controller/org.controller/error')
+const error = new errors.OrgControllerError()
+
+describe('Testing the /org/:shortname/users endpoint in Org Controller', () => {
+    before(async () => {
+
+    })
+
+    context('Negative Tests', () => {
+        it('should return 404 becuase org does not exist', (done) => {
+            class NoOrg {
+                async getOrgUUID () {
+                    return null
+                }
+            }
+        
+            class Blank {
+                async aggregate () {
+                    return []
+                }
+            }
+        
+            server.route('/org-does-not-exist/:shortname/users')
+              .get((req, res, next) => {
+                const factory = {
+                  getOrgRepository: () => { return new NoOrg() },
+                  getUserRepository: () => { return new Blank() }
+                }
+                req.ctx.repositories = factory
+                next()
+              }, orgParams.parseGetParams, orgController.USER_ALL)
+
+            // perform the request to the api
+            chai.request(server)
+                .get(`/org-does-not-exist/${orgFixtures.nonExistentOrg.short_name}/users`)
+                .set(orgFixtures.owningOrgHeader)
+                .end((err, res) => {
```

Comment:
Is the lack of indentation on the `end` block of each of these Chai requests intentional?

Reply:
Broad question, does linting enforce white space and 2- vs. 4-space tabs? It looks like some of these files use 2-space and some use 4-space

Reply:
I still haven't gotten VS code to properly follow the style of this codebase and I think different files follow different things... I hope once we get the linter in we just lent the whole thing and then I can get my editor set right

Reply:
That's where I was going with my second comment :)